src/unity/toolkits/activity_classification/ac_data_iterator.hpp

```diff
@@ -60,6 +60,8 @@ class data_iterator {
      * an exception will be thrown upon encountering an unexpected label.
      */
     flex_list class_labels;
+
+    bool is_train;
```

Comment:
Could we instead call this parameter `verbose`? Then the name better suggests what impact the value has. We can also set a default here.

Either way, let's add a comment documenting the parameter. (Follow the example of the other struct members here, which use Doxygen-style comments that can be automatically compiled into our C++ API docs.)